Sprint-2/debug/author.js

```diff
@@ -11,6 +13,10 @@ const author = {
   alive: true,
 };
 
-for (const value of author) {
+// for (const value of author) {
+//   console.log(value);
```

Comment:
Please remove dead code

Reply:
thank you for your review @mhmdk, I'v updated this code